.gitignore

```diff
@@ -48,5 +48,5 @@ all-links.csv
 standaloneContentBuildHash.txt
 websiteContentBuildHash.txt
 
-
+cms.diff
```

Comment:
I should know what this is, shouldn't I?

Reply:
It’s an artifact of the `yarn cam:compare` script.